test/e2e_test.go

```diff
@@ -557,8 +557,14 @@ func downloadTSACerts(downloadDirectory string, tsaServer string) (string, strin
 }
 
 func TestSignVerifyWithTUFMirror(t *testing.T) {
-	home, err := os.UserHomeDir() // fulcio repo was downloaded to $HOME in e2e_test.sh
-	must(err, t)
+	dir, err := os.Getwd()
+	if err != nil {
+		t.Fatal(err)
+	}
+	// Copied from https://github.com/sigstore/fulcio/blob/273116884c6e247a98cb28a9acc3e5844c4f9b5c/config/ctfe/pubkey.pem
+	// This must be an abosulte path.
+	fulcioPubkey := filepath.Join(dir, "testdata", "fulcio", "config", "ctfe", "pubkey.pem")
```

Comment:
Name should be `ct` rather than `fulcio`, since it's the CT log's public key.

Reply:
changed.